cli/flags.go

```diff
@@ -12,16 +12,28 @@ import (
 type Flags struct {
 	Topic     string
 	Namespace string
+	Host      string
 }
 
 type Credentials struct {
 	User     string
 	Password string
 }
 
+const HostEnv = "ZED_LAKE_HOST"
```

Comment:
Maybe use HostEnv from zed/cmd/zed/api/command.go?

Reply:
Yeah I copied this from there, but importing ould bring in a bunch of stuff that isn't used.  If we wanted to do this, we should move to this logic to `zed/cli/hostflags`